Assets/MRTK/Tests/TestUtilities/TestUtilities.cs

```diff
@@ -1,31 +1,29 @@
 ﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-using System.Collections.Generic;
-using System.Linq;
-using System.Reflection;
 using UnityEngine;
-using UnityEngine.SceneManagement;
 
 #if UNITY_EDITOR
 using Microsoft.MixedReality.Toolkit.Utilities.Editor;
 using Microsoft.MixedReality.Toolkit.Editor;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
 using UnityEditor;
 using UnityEditor.SceneManagement;
-#endif
-
-#if WINDOWS_UWP
-using UnityEngine.Assertions;
+using UnityEngine.SceneManagement;
 #endif
 
 namespace Microsoft.MixedReality.Toolkit.Tests
 {
     public static class TestUtilities
     {
-        const string primaryTestSceneTemporarySavePath = "Assets/__temp_primary_test_scene.unity";
-        const string additiveTestSceneTemporarySavePath = "Assets/__temp_additive_test_scene_#.unity";
-        public static Scene primaryTestScene;
```

Comment:
Is it possible some developers are using these?

Reply:
Hmm...I suppose it's possible, yeah! I'll rework this change

Reply:
Reverted the change for now